build/docker/pbs-cluster/Dockerfile

```diff
@@ -0,0 +1,94 @@
+FROM ubuntu:22.04
+
+# Set environment variables
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install required dependencies
+RUN apt-get update && apt-get install -y \
+    wget \
+    gcc \
+    make \
+    libssl-dev \
+    libtool \
+    libhwloc-dev \
+    libx11-dev \
+    libxt-dev \
+    libedit-dev \
+    libmunge-dev \
+    libpq-dev \
+    libical-dev \
+    swig \
+    tcl-dev \
+    munge \
+    vim \
+    curl \
+    tk-dev \
+    tk8.6-dev \
+    python3-dev \
+    python3 \
+    python3-pip \
+    wget curl vim git \
+    build-essential libtool autoconf automake \
+    libtool-bin g++ gcc \
+    libhwloc-dev libx11-dev libxt-dev \
+    libedit-dev libical-dev ncurses-dev \
+    perl python3-dev tcl-dev tk-dev swig \
+    libexpat-dev libssl-dev \
+    openssh-server \
+    postgresql postgresql-contrib \
+    supervisor \
+    sshpass \
+    && rm -rf /var/lib/apt/lists/*
+
+
+# Download and install PBS Pro Open Source
+WORKDIR /opt
+RUN wget -O pbspro.tar.gz https://github.com/openpbs/openpbs/archive/refs/tags/v23.06.06.tar.gz \
```

Comment:
can we parametrize the version "23.06.06" as an env var at the beginning of the Dockerfile?